server.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Request, Response } from 'express';
+import { renderFile } from 'ejs';
+
+const PORT = 80;
```

Comment:
Wdyt about adding types here even if it's redundant (and breaks no-inferrable-types or whatever that linting rule is called). I like it because it emphasises that this is TS and not JS.

Reply:
I would personally not add types in constants, especially primitives ones, unless it's a coding standard. It's quite straightforward to tell what this is... I do see some value in declaring the value of `express()` though (which I actually originally had and then removed), since it's not obvious what that is. So, added that back in :+1: .